File: simple-fun-number-74-growing-plant/simple-fun-number-74-growing-plant.py
import math
def growing_plant(upSpeed, downSpeed, desiredHeight):
    if upSpeed>desiredHeight:
        return 1
    x = (desiredHeight-upSpeed)/(upSpeed-downSpeed)
    return  math.floor(x)+2  if x-math.floor(x)>0 else math.floor(x)+1

File: simple-fun-number-74-growing-plant/simple-fun-number-74-growing-plant.js
function growingPlant(up, down, desiredHeight) {
  if(up>desiredHeight){return 1}
  var x = (desiredHeight-up)/(up-down)
  return x-Math.floor(x)>0 ?  Math.floor(x)+2 : Math.floor(x)+1
}